salt/modules/win_update.py

```diff
@@ -29,6 +29,7 @@
 # pylint: enable=import-error
 
 # Import salt libs
+from salt.utils.locales import sdecode
```

Comment:
It doesn't need to be. It works as is. It becomes available in both the salt.utils.locales and local namespaces. I can change it though. Give me a minute.